doctrine.rst

```diff
@@ -366,305 +166,305 @@ directly inside the ``Product`` class via DocBlock annotations:
             xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping
                 http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
 
-            <entity name="App\Entity\Product" table="product">
+            <entity name="App\Entity\Product">
                 <id name="id" type="integer">
                     <generator strategy="AUTO" />
                 </id>
-                <field name="name" type="string" length="100" />
-                <field name="price" type="decimal" scale="2" />
-                <field name="description" type="text" />
+                <field name="name" type="string" length="100" nullable="false" />
+                <field name="price" type="decimal" scale="2" nullable="false" />
             </entity>
         </doctrine-mapping>
 
-.. note::
+Doctrine supports a wide variety of different field types, each with their own options.
+To see a full list of types and options, see `Doctrine's Mapping Types documentation`_.
 
-    A bundle can accept only one metadata definition format. For example, it's
-    not possible to mix YAML metadata definitions with annotated PHP entity
-    class definitions.
+.. caution::
 
-.. tip::
+    Be careful not to use reserved SQL keywords as your table or column names
+    (e.g. ``GROUP`` or ``USER``). See Doctrine's `Reserved SQL keywords documentation`_
+    for details on how to escape these. Or, configure the table name with
+    ``@ORM\Table(name="groups")`` or the column name with the ``name="group_name"``
```

Comment:
what about to show directly how to escape keywords here? ``@ORM\Table(name="`group`")`` or the column name with the ``name="`group`"``